### ISSUE TITLE
error with DHL

### ISSUE THREAD

**Issue by radebold** [TA2k/ioBroker.parcel#43]
I received this error from DHL

`<html><body>

  | "<HTML><HEAD>\n<TITLE>Internal  Server  Error</TITLE>\n</HEAD><BODY>\n<H1>Internal  Server Error - Read</H1>\nThe server encountered an internal error  or misconfiguration and was unable to\ncomplete your  request.<P>\nReference&#32;&#35;3&#46;24257e68&#46;1680588028&#46;ad8edecc\n</BODY></HTML>\n"
-- | --

</body>
</html>`

**Comment by TA2k** [comment]
Einfach mal ein paar stunden warten Fehler 500 ist ein dhl Fehler

**Comment by Flothoger** [comment]
I can confirm the issue with login to DHL. I tried to wait a couple of days because I thought it might be a rate-limit but the issue seems to persist.

In the log you can see that there is also the AxiosError. So I guess #44 references the same issue.

```
parcel.1 | 2023-04-11 11:07:56.137 | error | "<HTML><HEAD>\n<TITLE>Internal Server Error</TITLE>\n</HEAD><BODY>\n<H1>Internal Server Error - Read</H1>\nThe server encountered an internal error or misconfiguration and was unable to\ncomplete your request.<P>\nReference&#32;&#35;3&#46;44d96b68&#46;1681204076&#46;d32431d\n</BODY></HTML>\n"
parcel.1 | 2023-04-11 11:07:56.137 | error | AxiosError: Request failed with status code 500
parcel.1 | 2023-04-11 11:07:55.811 | info | Login to DHL
parcel.1 | 2023-04-11 11:07:55.773 | info | starting. Version 0.0.25 in /opt/iobroker/node_modules/iobroker.parcel, node: v16.19.0, js-controller: 4.0.24
```

**Comment by TA2k** [comment]
is fixed with 0.0.27
